src/common/Tooltip/TooltipArea.tsx

```diff
@@ -284,7 +290,7 @@ export const TooltipArea = forwardRef<any, Partial<TooltipAreaProps>>(({
       valueScale = yScale;
     }
 
-    const newValue = getClosestPoint(coord, keyScale, transformed, 'x');
+    const newValue = isContinous ? getSelectedSegment(coord, transformed, 'x') : getClosestPoint(coord, keyScale, transformed, 'x');
```

Comment:
I wasn't 100% on the name of this prop `isContinous`. What it implies is that the chart has no breaks (aesthetically continous?). It differentiates how this `newValue` for the tooltip is grabbed.

`getClosestPoint` doesn't apply to FunnelCharts like it does to BarCharts. If the closest point is grabbed on a FunnelChart, as soon as you hover more than half way through a segment the tooltip will update to the next segment, so I built a new function `getSelectedSegment`

Reply:
Actually I think the bar chart tooltip is a bug - could we update the bar chart to use this and solve that bug?

Reply:
Yea I'll try that

Reply:
Ok I added support for FunnelCharts to use the getClosestPoint() util and I also fixed that bar chart tooltip bug